src/language-js/parser-typescript.js

```diff
@@ -69,6 +69,10 @@ const parser = Object.assign({ parse, astFormat: "estree", hasPragma }, locFns);
 module.exports = {
   parsers: {
     typescript: parser,
+    "typescript-estree": parser,
+    /**
+     * TODO: Remove this old alias in a major version
+     */
     "typescript-eslint": parser
```

Comment:
Actually I think we can remove this and just keep the alias @ikatyang linked:

```js
get "typescript-eslint"() {
  return eval("require")("../../whatever").typescript;
}
```

Who refactored this part (me) forgot this could be updated :-)

@ikatyang is this correct?